src/util/prolog/prologResult.ts

```diff
@@ -42,6 +52,10 @@ export class PrologResult {
 		return results;
 	}
 
+	/**
+		* Alternative way to access the results via an array.
+		* @return {Array<Array<string>>} First Level: i-1-th answer Second Level: Variable in query's order
```

Comment:
Nit: Isn't "i-1-st" correct?

Reply:
It would be correct. I meant the i-th answer though. So I accidentally added the 1 it seems.